src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1478,15 +1529,20 @@ export class ResultsViewPageStore {
         invoke: ()=>Promise.resolve(indexHotspotsData(this.hotspotData))
     });
 
-    public readonly isHotspot = remoteData({
+    public readonly isHotspot = remoteData<Error|((mutation:Mutation)=>boolean)>({
```

Comment:
Since we are not getting hotspot info from OncoKB anymore, changes related to hotspots will conflict with https://github.com/cBioPortal/cbioportal-frontend/pull/1083/files#diff-a1e9801197401cd10e07242602cf120dL1481.